experiment/coverage/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
```

Comment:
also reviewers please!

Reply:
ah, OK. I will review also.
Your review is so fast anyways :-)